diff --git a/training/nn.py b/training/nn.py
index 04e4d39..650074e 100644
--- a/training/nn.py
+++ b/training/nn.py
@@ -64,7 +64,8 @@ my_metrics = ['mae']
 n_epochs, batch_size = 3, 100
 
 network = models.Sequential()
-network.add(layers.Dense(X_train.shape[1], activation='relu', input_dim=X_train.shape[1]))
+network.add(layers.Dense(X_train.shape[1], activation='sgd', input_dim=X_train.shape[1]))
+network.add(Dropout(0.2, input_shape=(X_train.shape[1],)))
 for i in range(10):
     network.add(layers.Dense(128,activation='relu'))
 network.add(layers.Dense(y_train.shape[1],activation='linear'))
@@ -95,4 +96,4 @@ plt.title('Model loss')
 plt.ylabel('Loss')
 plt.xlabel('Epoch')
 plt.legend(['Train', 'Test'], loc='best')
-wandb.log({"loss": plt})
+wandb.log({"loss": plt})
\ No newline at end of file
